compiler_opt/rl/corpus.py

```diff
@@ -34,77 +35,93 @@ def build_modulespecs_from_datapath(
     additional_flags: Tuple[str, ...] = (),
     delete_flags: Tuple[str, ...] = ()
 ) -> List[ModuleSpec]:
-  module_paths: List[str] = _load_module_paths(data_path)
-
-  has_thinlto: bool = _has_thinlto_index(module_paths)
+  # TODO: (b/233935329) Per-corpus *fdo profile paths can be read into
+  # {additional|delete}_flags here
+  corpus_description: Dict[str, Any] = _load_corpus_description(
+      os.path.join(data_path, 'corpus_description.json'))
+  module_paths = corpus_description['modules']
+  if len(module_paths) == 0:
+    raise ValueError(f'{data_path}\'s corpus_description contains no modules.')
+
+  has_thinlto: bool = corpus_description['has_thinlto']
+
+  if 'global_command_override' in corpus_description:
+    if corpus_description['global_command_override'] == ['<UNSPECIFIED>']:
```

Comment:
sorry, one more nit: can you define a module constant for the `<UNSPECIFIED>` value, and then use that in extract_ir and elsewhere, rather than the string? it would help maintainability

Reply:
got it, I put it in `extract_ir.py` though, so that `extract_ir.py` can be run from whatever cwd without trying to import from `compiler_opt`

Reply:
slight problem: running `import extract_ir` from train_locally triggers the absl.FLAGS stuff, and we have a name conflict there due to `num_workers` (doing the reverse would be the same result)

Reply:
ah well, just moved it to constant.py